config/software/redis.rb

```diff
@@ -8,225 +8,12 @@
 relative_path "redis-2.8.17"
 
 etc_path = "#{install_dir}/embedded/etc"
+omnibus_flapjack_path = Dir.pwd
 
 make_args = ["PREFIX=#{install_dir}/embedded",
              "CFLAGS='-L#{install_dir}/embedded/lib -I#{install_dir}/embedded/include'",
              "LD_RUN_PATH=#{install_dir}/embedded/lib"].join(" ")
 
-config = <<-CONFIG
-daemonize yes
-pidfile /var/run/flapjack/redis-flapjack.pid
-port 6380
-bind 0.0.0.0
-timeout 300
-loglevel notice
-logfile /var/log/flapjack/redis-flapjack.log
-databases 16
-save 900 1
-save 300 10
-save 60 10000
-rdbcompression yes
-dbfilename dump.rdb
-dir /var/lib/flapjack/redis-flapjack
-slave-serve-stale-data yes
-appendonly no
-appendfsync everysec
-no-appendfsync-on-rewrite no
-CONFIG
-
-deb_init = <<-INIT
-#! /bin/sh
-### BEGIN INIT INFO
-# Provides:   redis-flapjack
-# Required-Start: $syslog $remote_fs
-# Required-Stop:  $syslog $remote_fs
-# Should-Start:   $local_fs
-# Should-Stop:    $local_fs
-# Default-Start:  2 3 4 5
-# Default-Stop:   0 1 6
-# Short-Description:  redis-flapjack - Persistent key-value db for Flapjack
-# Description:    redis-flapjack - Persistent key-value db for Flapjack
-### END INIT INFO
-
-
-PATH=#{install_dir}/embedded/bin:/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
```

Comment:
when moved to files we'll need to treat them as erb templates or some such so these vars are interpolated